RULES.md

```
@@ -0,0 +1,37 @@
+Rules file Copied from @Jmen, https://avatars.githubusercontent.com/u/5757596?v=4
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)